ascan

```diff
@@ -1,37 +1,38 @@
 AbsScanPlan([det], motor, 1, 5, 3)
-stage: (mover: motor), (), {}
-stage: (reader: det), (), {}
-open_run: (None), (), {'plan_type': 'AbsScanPlan', 'motors': ['motor'], 'plan_args': {'motor': 'mover: motor', 'start': '1', 'stop': '5', 'detectors': '[reader: det]', 'num': '3'}, 'detectors': ['det']}
-wait: (None), ('_flyers',), {}
+open_run: (None), (), {'detectors': ['det'], 'motors': ['motor'], 'plan_args': {'num': 3, 'detectors': '[reader: det]', 'stop': 5, 'start': 1}}
 checkpoint: (None), (), {}
-set: (mover: motor), (1.0,), {'block_group': 'A'}
-wait: (None), ('A',), {}
+stage: (mover: motor), (), {}
+set: (mover: motor), (1.0,), {'block_group': 'set-dc4e7d'}
+wait: (None), ('set-dc4e7d',), {}
 create: (None), (), {'name': 'primary'}
-trigger: (reader: det), (), {'block_group': 'B'}
-wait: (None), ('B',), {}
+stage: (reader: det), (), {}
+trigger: (reader: det), (), {'block_group': 'trigger-ad454c'}
+trigger: (mover: motor), (), {'block_group': 'trigger-ad454c'}
```

Comment:
Also, the motors should already be in position at this point. What does it mean to trigger them here?